tests/test_run_state_async.py

```
@@ -0,0 +1,57 @@
+    # Setup
+    manager = RunStateManager(runs_root=tmp_path)
+
+    # Create some dummy runs
+    for i in range(5):
+        run_id = f"run-{i}"
+        flow_id = f"flow-{i}"
+        state = {
+            "run_id": run_id,
+            "flow_id": flow_id,
+            "status": "completed",
+            "created_at": datetime.now(timezone.utc).isoformat(),
```

Ruff (0.14.14)

[warning] Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)